### PR TITLE
nvidia check: don't fail on max host on VM's

### PR DESCRIPTION
**Pull request by wdpypere**

### CODE REVIEW

File: scripts/csc_nvidia_smi.nhc [hunk 1]
```diff
@@ -79,7 +79,12 @@ function check_nvmsi_number() {
         return 1
     fi
 
-    unkn=$(echo "$NVSMI_HEALTHMON_OUTPUT" | grep -i 'Unknown Error' | wc -l)
+    if $(systemd-detect-virt 2>/dev/null); then
+        unkn=$(echo "$NVSMI_HEALTHMON_OUTPUT" | grep -i 'Unknown Error' | wc -l)
```
Comment (stdweird) [inline scripts/csc_nvidia_smi.nhc]: it's a bit too much copy paste to my liking.
maybe
```
filter="NOMATCH"
 if $(systemd-detect-virt 2>/dev/null); then
    filter="Host Max"
fi
unkn=$(echo "$NVSMI_HEALTHMON_OUTPUT" | grep -i 'Unknown Error' | grep -v "$filter" | wc -l)
```

Comment (wdpypere) [reply scripts/csc_nvidia_smi.nhc]: sure

File: lbnl-nhc.spec.in [hunk 2]
```diff
@@ -43,7 +43,7 @@
 Summary: LBNL Node Health Check
 Name: %{name}
 Version: %{version}
-Release: %{release}%{?dist}.ug
+Release: %{release}.ug.1
```
Comment (wdpypere) [inline lbnl-nhc.spec.in:46]: dist is already included in release. we now have a double el8 in `lbnl-nhc-1.4.3-0.155.g4343.el8.el8.ug.noarch`

Comment (wdpypere) [reply lbnl-nhc.spec.in:46]: and yes, that is a UGent change :smile:

Comment (wdpypere) [reply lbnl-nhc.spec.in:46]: not sure what yum will think of it though. if it will be regarder as higher.

Comment (wdpypere) [reply lbnl-nhc.spec.in:46]: @itkovian opinions?

Comment (itkovian) [reply lbnl-nhc.spec.in:46]: I would ditch the dist then.

**Pull request merged**